WORKSPACE

```diff
@@ -36,6 +36,12 @@ http_archive(
     sha256 = "886db2f8d620fcb5791c8e2a402a575bc70728e17ec116841d78f3837a09f69e",
     strip_prefix = "rules_go-9bb1562710f7077cd109b66cd4b45900e6d7ae73",
     urls = ["https://github.com/bazelbuild/rules_go/archive/9bb1562710f7077cd109b66cd4b45900e6d7ae73.tar.gz"],
+    patch_args = ["-p1"],
+    patches = [
+        # TODO remove patch once https://github.com/bazelbuild/rules_go/pull/2283 is merged
```

Comment:
Let's block this PR until https://github.com/bazelbuild/rules_go/pull/2283 is merged, if that's OK.